src/condor_collector.V6/ce-audit-plugin.cpp

```diff
@@ -45,7 +47,7 @@ CEAuditPlugin::stopJob(const ClassAd& ad) {
         matchRE = ".*";
     } else {
         // names of form "slotN@" stop that name and all "slotN_M@" names
-        Regex re; re.compile( "^(slot[0-9]*)@.*'", NULL, NULL );
+        Regex re; re.compile( "^(slot[0-9]*)@.*'", &errcode, erroffset );
```

Comment:
I think you forgot the pass by reference here for erroffset.

Reply:
Yep. And I didn't realize when making this PR that this file isn't built on mac. Fixing now...